keyboards/idobao/id42/keymaps/via/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 Vino Rodrigues (@vinorodrigues)
```

Comment:
This file should be removed completely, as this will no longer be needed at the end of the month (or sooner if you grab the binary from qmk.tzarc.io)